Goutte/Client.php

```diff
@@ -129,14 +129,16 @@ protected function doRequest($request)
                 $requestOptions['body'] = $content;
             } else {
                 if ($files = $request->getFiles()) {
-                    $requestOptions['multipart'] = [];
+                    $requestOptions['multipart'] = array();
```

Comment:
I have reverted the changes related to the coding standard test and left only the change related to the GET parameters problem. If you prefer, I could send a new pull request with that change only and forget this pull request.